ingest/build.gradle

```diff
@@ -21,7 +34,14 @@ dependencies {
     implementation(Dependencies.commonsIO())
     implementation(Dependencies.ingestAPI())
     implementation(Dependencies.transformAPI())
+
     testImplementation(Dependencies.springBootStarterTest())
+
+    integrationTestImplementation(Dependencies.ingestAPI())
+    integrationTestImplementation(Dependencies.transformAPI())
+    integrationTestImplementation(Dependencies.springBootStarterTest())
+    integrationTestImplementation(Dependencies.springBootStarterWeb())
+    integrationTestImplementation(Dependencies.reactiveStreams())
```

Comment:
❓Shouldn't the itests inherit all of the dependencies from `implementation`?

Reply:
That's what I thought, but it seems like it's not the case

Reply:
I think we should figure out how to make that work. I could see an issue where we don't maintain the dependencies between implementation and itestImplementation, which means that we're running the itests with different dependencies.

Reply:
This was a nightmare to look into, I think it should be a separate pr.

Reply:
I'm not okay merging until we figure this out. I think this change introduces a lot of risk.

Reply:
CC @lessarderic